grakn-engine/src/main/java/ai/grakn/engine/controller/GraqlController.java

```diff
@@ -137,9 +156,9 @@ private Object executeGraql(Request request, Response response) {
             return resp;
         }
     }
-    
+
     @GET
-    @Path("/")
+    @Path("/graph/graql")
```

Comment:
I don't get why we need this endpoint? What is it actually used for?

Reply:
This is a deprecated endpoint that returns things in a very particular format that the HAL representation needs. @marco-scoppetta is working on removing it.